api/users/serializers.py

```diff
@@ -119,6 +125,16 @@ def get_can_view_reviews(self, obj):
         group_qs = GroupObjectPermission.objects.filter(group__user=obj, permission__codename='view_submissions')
         return group_qs.exists() or obj.userobjectpermission_set.filter(permission__codename='view_submissions')
 
+    def get_default_region_id(self, obj):
+        try:
+            # use the annotated valud if possible
+            region = obj.default_region
+        except AttributeError:
```

Comment:
Is this error handling necessary? All users should have a default region.

Reply:
`UserSettings` objects have a `default_region`, but `OSFUser` objects do not (unless it's been [annotated](https://github.com/CenterForOpenScience/osf.io/pull/8401/files#diff-dd88c3f184d385cd8bef57c939c6d955R69)).